inc/composer/class-command.php

```diff
@@ -654,6 +661,179 @@ protected function db( InputInterface $input, OutputInterface $output ) {
 		return $return_val;
 	}
 
+	/**
+	 * Generate SSL certificates for development environment.
+	 *
+	 * @param InputInterface $input Command input object.
+	 * @param OutputInterface $output Command output object.
+	 * @return int
+	 */
+	protected function ssl( InputInterface $input, OutputInterface $output ) {
+		$subcommand = $input->getArgument( 'options' )[0] ?? null;
+
+		switch ( $subcommand ) {
+			case 'install':
+				// Detect platform architecture to attempt automatic installation.
+				$os = php_uname( 's' ); # 'Darwin', 'Linux', 'Windows'
+				$arch = php_uname( 'm' ); # 'arm64' for arm, 'x86_64' or 'amd64' for x64
+				$mkcert_version = 'v1.4.3';
+
+				switch( $os ) {
+					# macOS
+					case 'Darwin':
+						$binary_arch = $arch === 'x86_64' ? 'darwin-amd64' : 'darwin-arm64';
+						break;
+					# Linux
+					case 'Linux':
+						$binary_arch = $arch === 'amd64' ? 'linux-amd64' : 'linux-arm64';
+						break;
+					# Windows
+					case 'Windows':
+						$binary_arch = 'windows-amd64.exe';
+						break;
+					default:
+						$binary_arch = null;
+						break;
+				}
+
+				// If couldn't detect a support architecture, ask the user to install mkcert manually.
+				if ( ! $binary_arch ) {
+					$output->writeln( '<error>This command is only supported on macOS, Linux, and Windows x64, install `mkcert` manually for other systems.</error>' );
+					return 1;
+				}
+
+				$binary = "mkcert-$mkcert_version-$binary_arch";
+				$mkcert = "vendor/mkcert";
+
+				// Check if mkcert is installed globally already, bail if so.
+				$version = trim( shell_exec( 'mkcert -version' ) );
+				if ( $version ) {
+					$output->writeln( "<error>mkcert $version is installed globally already</error>" );
```

Comment:
Does it have a way to check if the root CA is already set up? Bit edge case but there's a chance someone might not have done that yet. I think that can probably be covered by an FAQ entry though.

Reply:
The output complains clearly, but we can probably script it as well by checking the output itself.

Reply:
I think that's fine then. Lets make improvements later if we find things are proving tricky for people.